src/StoryBoard.cpp

```diff
@@ -457,23 +482,26 @@ bool StoryBoard::runFunct(unsigned int fID, NaughtyFilter &cm) {
                 state_result = true;
                 break;
         }
-#ifdef SBDEBUG
-        std::cerr << "SB state " << F->getState(i->state) << " target " << target << " target2 " << target2
-                  << " state_result " << state_result <<
-                  " list_check " << isListCheck << " targetfull " << targetful << " isSearch " << cm.isSearch
-                  << std::endl;
-#endif
+
+        DEBUG_story("SB-Test  ",
+                    " state: ", F->getState(i->state),
+                    " target: ", target,
+                    " target2: ", target2,
+                    " list_check: ", String(isListCheck),
+                    " targetfull: ", targetful,
+                    " isSearch: ", String(cm.isSearch),
+                    " state_result: ", String(state_result));
 
         if (isHeaderCheck) {
-            for (std::deque<String>::iterator u = cm.request_header->header.begin();
-                 u != cm.request_header->header.end(); u++) {
+            DEBUG_trace("HeaderCheck");
+            for (std::deque<String>::iterator u = targetheader->header.begin();
+                 u != targetheader->header.end(); u++) {
```

Comment:
Codacy found an issue: [Prefer prefix ++/-- operators for non-primitive types.](https://app.codacy.com/manual/numsys/e2guardian/pullRequest?prid=6238852)